### PR TITLE
is string empty

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/pcgen/io/exporttoken/EqToken.java [hunk 1]
```diff
@@ -248,7 +248,7 @@ else if ("IS".equalsIgnoreCase(token))
 							aTok));
 				 
 				// Starting EQ.%.NAME.MAGIC,befTrue,aftTrue,befFalse,aftFalse treatment
-				if (!"".equals(bFilter))
+				if (bFilter != null && !bFilter.isEmpty())
```
Comment (thpr) [inline code/src/java/pcgen/io/exporttoken/EqToken.java]: parens around the null checks please

File: code/src/java/pcgen/io/exporttoken/FollowerToken.java [hunk 2]
```diff
@@ -118,7 +118,8 @@ public static String getFollowerOutput(ExportHandler eh,
 		StringWriter writer = new StringWriter();
 		BufferedWriter bw = new BufferedWriter(writer);
 
-		String token = "".equals(followerToken) ? "NAME" : followerToken;
+		String token =
+				followerToken != null && followerToken.isEmpty() ? "NAME" : followerToken;
```
Comment (thpr) [inline code/src/java/pcgen/io/exporttoken/FollowerToken.java]: and here

**Comment by grimreaper** [comment]
resolved parens

**Comment by grimreaper** [comment]
dup of #2526

**Pull request closed without merging**